ios/KMMIC/Sources/Supports/Extensions/SwiftUI/View+Alert.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  View+Alert.swift
+//  KMMIC
+//
+//  Created by MarkG on 20/04/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import Foundation
+import Shared
+import SwiftUI
+
+struct AlertDescription: Identifiable, Equatable {
```

Comment:
We could separate this into another file

Reply:
@phongvhd93 updated at [d74bf92](https://github.com/markgravity/kmm-ic/pull/69/commits/d74bf9208aaaaeda1de406a04b09cde4dfd9f9f1)